docs/tsg_install.md

```diff
@@ -136,8 +136,55 @@ kubectl delete crd volumesnapshotdatas.volumesnapshot.external-storage.k8s.io
 kubectl delete crd volumesnapshots.volumesnapshot.external-storage.k8s.io
 ```
  
+## cStor pool creation fails on Cetos when there are partitions on the disk 
```

Comment:
Cetos = CentOS
Creating cStor pool fails on CentOS when there are partitions on the disk